crates/bevy_core/src/time/fixed_timestep.rs

```diff
@@ -1,301 +1,436 @@
-use crate::Time;
 use bevy_ecs::{
-    archetype::{Archetype, ArchetypeComponentId},
-    component::ComponentId,
-    query::Access,
     schedule::ShouldRun,
-    system::{IntoSystem, Res, ResMut, System},
-    world::World,
+    system::ResMut,
+    world::{FromWorld, World},
 };
-use bevy_utils::HashMap;
-use std::borrow::Cow;
+use bevy_utils::{Duration, Instant};
 
-/// The internal state of each [`FixedTimestep`].
-#[derive(Debug)]
-pub struct FixedTimestepState {
-    step: f64,
-    accumulator: f64,
+use crate::Time;
+
+/// The default step size used by [`FixedTimestep`].
+// 60Hz is a popular tick rate, but it can't be expressed as an exact float.
+// The nearby power of two, 64Hz, is more stable for numerical integration.
+pub const DEFAULT_TIMESTEP: Duration = Duration::from_micros(15625); // 64Hz
```

Comment:
I think this may be easier to discover as a constant within a `FixedTime` impl block, since then it'll show up in the docs.